frontend/simple/main.js

```diff
@@ -10,54 +11,91 @@ import utils, { wrap, lazyLoadVue, superagentHeader } from './js/utils'
 import store from './js/state'
 import './js/transitions'
 
-Vue.config.debug = process.env.NODE_ENV === 'development'
 Vue.use(Router)
-Vue.use(require('vue-form'), {invalidClass: 'is-danger'})
+Vue.use(VeeValidate)
 
 superagentHeader('Authorization', `gi ${utils.sign('hello', utils.keypair)}`)
 
 var router = new Router({
-  hashbang: false,
-  history: true,
-  root: '/simple'
-})
-
-router.map({
-  '/': { component: SignUp },
-  '/signup': {
-    title: 'Sign Up',  // page title. see issue #45
-    name: SignUp.name, // route name. important!
-    component: SignUp
-  },
-  '/new-group': {
-    title: 'Create Group',
-    name: CreateGroup.name,
-    component: CreateGroup
+  mode: 'history',
+  base: '/simple',
+  scrollBehavior (to, from, savedPosition) {
+    return { x: 0, y: 0 }
```

Comment:
Check line 52 of the older code. This was created to emulate that functionality.